unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Loader/RetrieverFactory.cs

```diff
@@ -0,0 +1,8 @@
+namespace AvatarSystem
+{
+    public class RetrieverFactory : IRetrieverFactory
```

Comment:
Name too generic imo

Reply:
It is under the `AvatarSystem` namespace which vastly concretes its use. 

I couldn't find a more specific name, `WearableRetrieverFactory` could imply that the facial features are not handled by this factory.

Reply:
> @Suduck [#1596 (comment)](https://github.com/decentraland/unity-renderer/pull/1596#discussion_r788594394) The colors are set in the material, are you sure that would make it more readable?


If you feel that it is more readable this way you can let it be!